backend/Dockerfile

```diff
@@ -10,8 +10,10 @@ RUN go get -tags netgo \
 		github.com/golang/lint/golint \
 		github.com/kisielk/errcheck \
 		gopkg.in/mvdan/sh.v1/cmd/shfmt \
+		github.com/fatih/hclfmt \
 		github.com/mjibson/esc \
 		github.com/client9/misspell/cmd/misspell && \
+	chmod a+wr --recursive /usr/local/go/pkg && \
```

Comment:
This is necessary to allow `go install net` (https://github.com/weaveworks/scope/pull/2225/commits/f850f4486aa37639edd6eb8b86e21e64b60ef9b6)